.stylelintrc

```diff
@@ -0,0 +1,43 @@
+{
+  "rules": {
+    "color-no-invalid-hex": true,
+    "font-family-no-duplicate-names": true,
+    "font-family-name-quotes": ["always-where-recommended", { "severity": "warning" }],
+    "function-name-case": "lower",
+    "function-url-no-scheme-relative": true,
+    "function-url-quotes": ["always", { "severity": "warning" }],
+    "number-no-trailing-zeros": [true, { "severity": "warning" }],
+    "length-zero-no-unit": [true, { "severity": "warning" }],
+    "unit-case": "lower",
+    "unit-no-unknown": true,
+    "property-case": "lower",
+    "property-no-unknown": true,
+    "keyframe-declaration-no-important": true,
+    "declaration-no-important": true,
```

Comment:
Ah, that makes sense. I was envisioning some kind of separate "ignore" list we'd have to maintain, which sounds like a bad idea. But just some special comment at the point of instance is much nicer. So in that case I also vote for keeping this as a full-blown error; `!important` should only be allowed under very special circumstances.